spanner/sessionclient.go

```diff
@@ -404,21 +403,11 @@ func (sc *sessionClient) nextClient() (*vkit.Client, error) {
 		// Pick a grpc.ClientConn from a regular pool.
 		clientOpt = option.WithGRPCConn(sc.connPool.Conn())
 	}
-	client, err := vkit.NewClient(context.Background(), clientOpt)
+	client, err := newGRPCSpannerClient(context.Background(), sc, clientOpt)
 	if err != nil {
 		return nil, err
 	}
-	clientInfo := []string{"gccl", internal.Version}
-	if sc.userAgent != "" {
-		agentWithVersion := strings.SplitN(sc.userAgent, "/", 2)
-		if len(agentWithVersion) == 2 {
-			clientInfo = append(clientInfo, agentWithVersion[0], agentWithVersion[1])
-		}
-	}
-	client.SetGoogleClientInfo(clientInfo...)
-	if sc.callOptions != nil {
-		client.CallOptions = mergeCallOptions(client.CallOptions, sc.callOptions)
-	}
+
```

Comment:
Where do we move this to? How can we make sure `gccl` is not missing? Do we have any test coverage for this?

Reply:
In the existing tests we are asserting gccl and other related headers using grpcHeaderChecker, so it will ensure that new client also exports them.